### PR TITLE
Fix docker-compose.yml so pull and build commands works properly

### PR DESCRIPTION
**Pull request by knorrest**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.2'
 
 services:
     reverseproxy:
-        image: reverseproxy
+        build: docker-nginx
```
Comment (jumarko) [inline docker-compose.yml:5]: This means it's no longer using our Nginx configuration (redirects and stuff).

Comment (knorrest) [reply docker-compose.yml:5]: How do you mean? This means it builds the image using the docker-nginx subdirectory when doing `docker-compose up` or `docker-compose build`, and does not try to pull a nonexisting image on `docker-compose up` or `docker-compose pull`.

Comment (knorrest) [reply docker-compose.yml:5]: Could be that I missed something in `docker-nginx/Dockerfile`, I'll check that

Comment (knorrest) [reply docker-compose.yml:5]: Nope, this is correct

**Pull request merged**